src/cheri/rvy32-encoding.adoc

```diff
@@ -0,0 +1,304 @@
+|===
+| <<CLRPERM>> Rule | Permission            | Valid only if
+| RV32-l1-1        | <<c_perm>>            | <<r_perm>> (supersedes <<perm_req:base:c:r-or-w>>)
+| RV32-l1-2        | <<x_perm>>            | <<r_perm>>
+| RV32-l1-3        | <<w_perm>>            | not(<<c_perm>>) or <<lm_perm>>
+| RV32-l1-4        | <<x_perm>>            | <<w_perm>> or <<c_perm>>
+| RV32-l1-5        | <<lm_perm>>           | <<c_perm>> (supersedes <<perm_req:base:lm:c-and-r>>)
+| RV32-l1-6        | <<lm_perm>>           | <<w_perm>> or <<zylevels1_lg_perm>>
+| RV32-l1-7        | <<zylevels1_lg_perm>> | <<lm_perm>> (supersedes <<perm_req:zylevels1:lg:c-and-r>>)
+| RV32-l1-8        | <<zylevels1_sl_perm>> | <<lm_perm>> and <<w_perm>> (supersedes <<perm_req:zylevels1:sl:c-and-w>>)
+| RV32-l1-9        | <<x_perm>>            |
+  (<<c_perm>> and <<lm_perm>> and <<zylevels1_lg_perm>> and <<zylevels1_sl_perm>>) or +
+  (<<c_perm>> and <<lm_perm>> and <<zylevels1_lg_perm>> and not <<w_perm>>) or +
+  not (<<c_perm>> or <<lm_perm>> or <<zylevels1_lg_perm>> or <<zylevels1_sl_perm>>)
+| RV32-l1-10       | <<asr_perm>>          | <<w_perm>> and <<c_perm>> and <<x_perm>> (supersedes <<perm_req:base:asr:x>>)
+| RV32-l1-11       | <<m_bit>>             | <<x_perm>> and {cheri_default_ext_name} is implemented
+|===
+
+For RV32, the encodings which have the <<m_bit>> set to {int_mode_value} for {cheri_int_mode_name}
+are only valid if {cheri_default_ext_name} is implemented.
+Otherwise those encodings represent invalid permissions.
+
+[#M-field-encoding32, reftext="M-field encoding"]
+==== Capability Mode (M) Encoding
+
+The <<m_bit>> is encoded as bit zero of the <<AP-field-encoding32>> for the executable quadrant and only assigned meaning when the implementation supports {cheri_default_ext_name} (_and_ <<x_perm>> is set).
+In valid capabilities, the bit assigned to the <<m_bit>> must be zero if <<x_perm>> isn't set.
+
+[#SDP-field32, reftext="SDP-field"]
+==== Software-Defined Permissions (SDP) Encoding
+
+The <<SDP-field32>> is {cap_rv32_sdp_width} bits wide.
+The value of the <<SDP-field32>> bits of the <<GCPERM>> result maps 1:1 to the <<SDP-field32>> in the capability.
+
+NOTE: Software is completely free to define the usage of these bits.
+
+[#section_cap_sealed32]
+==== Capability Type (CT) Encoding
+
+Capabilities in this encoding have a 1-bit field for <<sec_cap_type>> values which behaves in the same way as the {rvy64_uni_base_name} <<section_cap_sealed>>.
+
+[#section_cap_bounds32]
+==== Bounds (EF, T, TE, B, BE, L8) Encoding
+
+The bounds are encoding in the same way as in <<section_cap_bounds,{rvy64_uni_base_name}>>, with the appropriate values for mantissa width and maximum exponent substituted.
+Compared to {rvy64_uni_base_name}, this encoding uses an additional `L8` bit as described in xref:section_cap_bounds[xrefstyle=short].
+
+===== <<AUIPC_CHERI>> Shift
+
+For the capabilities of this encoding, the <<section_auipc_shift>> value is 12.
```

Comment:
I'd note that the value 12 is unchanged from the default, and that other encoding formats may need to change it due to retaining the representability.